src/js/events/inithero.js

```diff
@@ -0,0 +1,102 @@
+import { $ } from '../utils/dom.js';
+
+let slideId;
+let index = 1;
+
+const getSlides = () => document.querySelectorAll('.slide-item');
```

Comment:
화살표함수도 조금씩 사용해보고 계시는군요. 좋습니다 ㅎㅎ

여기서 드리고 싶은 말씀은 두가지인데요!
1. 어떤 경우에 화살표 함수를 쓰고 어떤 경우에는 function 키워드를 사용할지 정리해보고 일관성 있게 작성해보시면 좋겠습니다!
2. 이 정도의 로직은 함수로 추출하지 않아도 괜찮을 것 같아요~
```suggestion
const slides = document.querySelectorAll('.slide-item');
```
추가적인 로직이 없어서 바로 변수에 담을 수 있으니까요 !

Reply:
- 한줄로 작성하면 좋겠다 싶어서 여기는 화살표함수로 바꿔주었어요 this가 달라지는 문제 등 일반 함수와 차이가 있다고 들었습니다 지금은 작성하는 저의 편의성이 우선시되었는데 어떤 경우에 나누어서 쓸 수 있는지, 화살표함수를 도입하면 안되는 경우들을 틈틈이 찾아보도록 하겠습니다!
- clone하기전과 clone후 캐러셀 slide요소의 개수가 달라져 함수로 바꿔주었습니다..!